dnf-behave-tests/features/microdnf/install_weak_deps.feature

```diff
@@ -0,0 +1,26 @@
+@no_installroot
+Feature: Tests --setopt=install_weak_deps=
+
+
+Background: Prepare repo
+  Given I delete file "/etc/dnf/dnf.conf"
+    And I delete file "/etc/yum.repos.d/*.repo" with globs
+    And I use repository "dnf-ci-fedora"
+
+
+Scenario: Install "abcde" without weak dependencies
+   When I execute dnf with args "install --setopt=install_weak_deps=0 abcde"
+   Then the exit code is 0
+    And RPMDB Transaction is following
```

Comment:
There is PR https://github.com/rpm-software-management/ci-dnf-stack/pull/731 with a step to check microdnf transaction properly. I'd prefer to merge that first and then use this step here.